Update GOVERNANCE.md (#38)
woodpecker-ci/.github: GOVERNANCE.md

- 2019-07-25 ~ 2021-09-28
  - [Laszlo Fogas](https://github.com/laszlocph)

## Access

> The project has multiple accounts and systems. Owners are granted access to all systems. Additional access might be provided to maintainers.

Reviewer: ```suggestion
The project has multiple accounts and systems. Owners have access to all systems. Maintainers may be granted additional access.
```

### Accounts

- [Github Org](https://github.com/woodpecker-ci/)
- [Codeberg Org](https://codeberg.org/woodpecker-ci)
- [Codeberg Plugins Org](https://codeberg.org/woodpecker-plugins)
- [OpenCollective](https://opencollective.com/woodpecker-ci)
- [Docker](https://hub.docker.com/u/woodpeckerci)

### Shared Accounts

Some accounts do require to share passwords. Those password should be shared using our [Vault](https://vault.woodpecker-ci.org).

- BlueSky
- Mastodon

> - YouTube

Reviewer 2: youtube account was ment ot make some marketing/adverticing/tutorial/teaching videos

but never got around it ... i would not delete it ... but we might wana outsource such things?

> - YouTube

Reviewer 2: fin an enthusiastic user in our comunity, that likes to and can,  do som videos

or setup smal budget as there are tech youtubers etc

> - YouTube

Reviewer 2: ack!

- GitHub Bot account

> - Codeberg Bot

Author: Ignored them as they are less critical. I would only include important accounts, otherwise the list is likely to be come outdated.

> ### Infrastructure

Reviewer 2: whatever needs less maintinance - mail is a pain in the ass!

- Domain: woodpecker-ci.org
  - owned by: [6543](https://github.com/6543)

>   - access: [Anbraten](https://github.com/anbraten)

Reviewer 2: @qwerty287 I'd like to have you there too as you are owner but somehow porkbun account did now work for you?!?

- 2019-07-25 ~ 2021-09-28
  - [Laszlo Fogas](https://github.com/laszlocph)

## Access

The project has multiple accounts and systems. Owners have admin access to all systems. Maintainers may be granted additional access.

### Accounts

- [GitHub Org](https://github.com/woodpecker-ci/)
- [Codeberg Org](https://codeberg.org/woodpecker-ci)
- [Codeberg Plugins Org](https://codeberg.org/woodpecker-plugins)
- [OpenCollective](https://opencollective.com/woodpecker-ci)

> - [Docker](https://hub.docker.com/u/woodpeckerci)

Reviewer 2: ```suggestion
- [Docker](https://hub.docker.com/u/woodpeckerci)
- [QuayIO](https://quay.io/organization/woodpeckerci)
```

Reviewer 2: <img width="1601" height="1440" alt="image" src="https://github.com/user-attachments/assets/861a9577-f3de-49d1-bfad-1db1c9b39033" />
<img width="1196" height="668" alt="image" src="https://github.com/user-attachments/assets/3d4f5a7f-f52b-411d-b631-ce4412ceafca" />

Reviewer 2: NOTE: the quai repo is as backup to not be threatened by `docker inc.` if they at some point want to do shit again
